tests/onnx/test_onnx_v2.py

```diff
@@ -499,6 +500,7 @@ def test_tensorflow_export_seq2seq_with_past(
 class StableDropoutTestCase(TestCase):
     """Tests export of StableDropout module."""
 
+    @unittest.skip("torch 2.0.0 gives `torch.onnx.errors.OnnxExporterError: Module onnx is not installed!`.")
```

Comment:
I realise this is more of a torch issue - but perhaps we want to open an issue to keep track and update this?